Makefile

```diff
@@ -22,7 +22,7 @@ docs: $(DIST)
 	cp -r $@ $^
 
 javascript: $(DIST)
-	@babel -d $^ $(JS)
+	@babel --plugins babel-plugin-unassert -d $^ $(JS)
```

Comment:
Super handy. I'm using `assert` to unit test hard to reach cases, stripping it out for production builds.